slsDetectorServers/moenchDetectorServer/slsDetectorFunctionList.c

```diff
@@ -1411,47 +1411,73 @@ int setReadoutSpeed(int val) {
     // stop state machine if running
     if (runBusy()) {
         stopStateMachine();
+        return FAIL;
+    }
+    if (isHardwareVersion_1_0()) {
+        LOG(logERROR, ("Cannot set full speed. Should not be here\n"));
```

Comment:
Is this the correct message?

Reply:
Ill change it to "Not implemented yet" or something